tests/Fixtures/ci-integration/setUp.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+cd $( cd -P "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+
+git init -q
+git config user.name test
+git config user.email test
+git add .
+git commit -m "init" -q
+
+git checkout -b case1 -q
```

Comment:
by adding more and more branches, it's possible to create next and next test cases